.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: php
 
 php:
-  - 7.2
```

Comment:
I think we shouldn't lose PHP 7.2 yet.

it would be good to keep it an additional time, until we migrated all repositories.

Reply:
Well thought Mario, I'll bring it back